editor/js/Sidebar.Material.js

```diff
@@ -642,7 +642,7 @@ Sidebar.Material = function ( editor ) {
 			'emissive': materialEmissiveRow,
 			'specular': materialSpecularRow,
 			'shininess': materialShininessRow,
-			'program': materialProgramRow,
+			'vertexShader': materialProgramRow,
```

Comment:
Yes, certainly. The program property is not defined right away and therefore the material editor won't show the program buttons until reloaded. The vertexShader property on the other hand is defined immediately when the visibility is determined.